services/hanke-service/src/main/resources/email/template/johtoselvitys-valmis.html.mustache

```diff
@@ -1,83 +1,225 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
-        "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html lang="fi">
-<body>
-<p>Hakemanne johtoselvitys {{applicationIdentifier}} on käsitelty. Lataa selvitys Haitattomasta (<a
-        href="{{baseUrl}}/fi/hakemus/{{applicationId}}">{{baseUrl}}/fi/hakemus/{{applicationId}}</a>) ja tutustu sen
-    sisältöön huolellisesti.</p>
+<!doctype html>
```

Comment:
Maybe we should keep this file as the old HTML version until we get approval for the new design. We can keep the mjml-file, just not compile it until we're ready to deploy it to production.

Reply:
Reverted.